packages/data-point/README.md

```diff
@@ -1354,6 +1355,74 @@ dataPoint.addEntities({
 | *EntityType* | `string` | valid entity type to associate with the EntityObject |
 | *EntityId* | `string` | unique entity ID associated with the EntityObject |
 
+### <a name="entity-factories">Entity factories</a>
+
+Entities can be defined with object literals, *or* with factories that DataPoint exposes. Each entity type has a corresponding factory:
+
+```js
+const {
+  Entry,
+  Model,
+  Reducer,
+  Collection,
+  Hash,
+  Request,
+  Control,
+  Schema
+} = require('data-point').entityFactories
+```
+
+
+The following examples are equivalent:
+
+**Example #1 (with object literal)**
+
+```js
+// object literal version
+dataPoint.addEntities({
+  'hash:hello-world': {
+    value: input => ({
+      hello: 'world'
+    })
+  }
+})
+```
+
+**Example #2 (with factory and assignment)**
+
+```js
+const { Hash } = DataPoint.entityFactories
+
+const hash = Hash({
```

Comment:
might be missing the id of the hash?

Reply:
good catch. fixed this and made sure these examples actually work correctly when executed.